app/package.json

```diff
@@ -13,7 +13,7 @@
   },
   "dependencies": {
     "@kibalabs/core": "0.5.7-next.1",
-    "@kibalabs/core-react": "^0.7.1-next.0",
+    "@kibalabs/core-react": "^0.7.1-next.2",
```

Comment:
i think most of these can be upgraded - i pushed new versions to all kiba libraries last week